charts/values.yaml

```diff
@@ -60,6 +60,13 @@ auth:
   adminPassword: ""
   walletPassword: ""
 
+# Optionally install converters package on MarkLogic
+installConverters: false
```

Comment:
It's generally suggested to design the boolean flags like:
converters:
    enabled: false
For two reasons:
1. It's easier to extend more feature in the future.
2. enabled is common practice to used as flag with true/false.

Reply:
@pengzhouml are you suggesting it be like

```
converters:
  enabled: false
```
I am ok with that there are no other converter configuration parameters that you can specify and I don't see that changing.

I did wonder about the license and if it should be something like but I didn't have a strong oppinion:
```
license:
  key:
  licensee:
```

Reply:
@jkerr5 Right. That what I was suggesting. Even the converters have no more options, it's still preferable to use "enabled" keyword as switch globally to turn features on/off. 
I do agree to group the license key and license together under license. Good Catch.

Reply:
It shouldn't just be `enableConverters`? We also have `enableXdqpSsl`. Just looking for some consistency.

Reply:
enableConverters should also be fine.

Reply:
That is fine with me too.